os/arch/arm/src/amebasmart/amebasmart_idle.c

```diff
@@ -48,7 +48,6 @@
  ****************************************************************************/
```

Comment:
This change seems to correspond to the following commit contents.
Could you check?